04-tests.bs

```diff
@@ -0,0 +1,128 @@
+<pre class='metadata'>
+Title: TREE client tests
+Shortname: tests
+Level: 1
+Status: w3c/CG-DRAFT
+Markup Shorthands: markdown yes
+URL: https://w3id.org/tree/specification/tests
+Repository: https://github.com/treecg/specification
+Mailing List: public-treecg@w3.org
+Mailing List Archives: https://lists.w3.org/Archives/Public/public-treecg/
+Editor: Pieter Colpaert, https://pietercolpaert.be
+Abstract:
+	A set of tests for testing compliance of clients to the TREE specification.
+</pre>
+
+# Tests # {#overview}
+
+## Initialization ## {#1}
+
+Purpose: Ensuring that it is unambiguous yet user friendly way of knowing the identifier of the collection, as well as the current page of the collection’s view.
+
+This test is about a function `initialization(url)`. The function will return a the collection and the URL of the root node.
+
+We’ll test this with various URLs containing different contents.
+
+### The URL to a rootnode without redirects ### {#1.1}
+
+File url `test1.1.ttl`
+
+```turtle
+@prefix tree: <https://w3id.org/tree#> .
+<Collection> tree:view <test1.1.ttl> .
+```
+
+Returns:
+ * Collection: `<Collection>`
+ * Root node: `<test1.1.ttl>`
+
+### The URL to a collection without redirects ### {#1.2}
+
+File url `test1.2.ttl#collection`
+
+```turtle
+@prefix tree: <https://w3id.org/tree#> .
+<test1.2.ttl#collection> tree:view <test1.2.ttl> .
```

Comment:
I don't think it is necessarily wrong, but wouldn't it be an idea to improve clarity by always using the uppercased`<testX.X.ttl#Collection>` for a Collection, instead of once `<Collection>` and the other time `<test1.2.ttl#collection>`. Similarly we could opt for `<testX.X.ttl#Node>` as Node URL, then we also test that it correctly returns the URL to the Node, and not just the URL to the RDF Document (as I think this is the intended behaviour).

Reply:
Sure!

Reply:
Solved this